pyEpiabm/pyEpiabm/core/microcell.py

```diff
@@ -171,3 +174,17 @@ def set_location(self, loc: typing.Tuple[float, float]):
                 isinstance(loc[1], Number)):
             raise ValueError("Location must be a tuple of float-type")
         self.location = loc
+
+    def count_icu(self):
+        num_icu = 0
+        for person in self.persons:
+            if person.infection_status == InfectionStatus.InfectICU:
+                num_icu += 1
+        return num_icu
```

Comment:
```suggestion
        return sum(map(lambda person: person.infection_status == InfectionStatus.InfectICU, self.persons))
```
You may need a new line in this for linting but I think it is much more pythonic than what you have right now, and because it uses an iterator it will likely be quicker too.